recipe/yum_requirements.txt

```diff
@@ -1,6 +1,6 @@
-mesa-libGL-devel
-mesa-libGLU-devel
-mesa-libEGL-devel
+libselinux-devel
+libXdamage
+libXxf86vm
```

Comment:
You may need libglvnd-devel here to fix the testing issues.

Reply:
Do we need the `-devel` or is there a non`-devel` version with the libraries?

Guessing we will also need the CDT as a test requirement

Reply:
Unfortunately, `libglvnd-devel` isn't available. Somehow `yum` cannot find it on the CI's VM, while on my local CentOS machine it works just fine...

Reply:
Can I add the CDT as test requirement instead?

Reply:
Yeah let's try that

Reply:
It worked :)